.github/ISSUE_TEMPLATE/4-release.yml

```diff
@@ -0,0 +1,123 @@
+
+  - type: markdown
+    attributes:
+      value: |
+        ## Checklist
+
+  - type: markdown
+    attributes:
+      value: |
+        ### RC release
+        
+        #### Get release artifact ready
+
+        -    [] Ensure minimally required WordPress and PHP versions are up-to-date
+        -    [] Ensure all [CFEs](https://github.com/woocommerce/woocommerce/issues?q=is%3Aissue%20state%3Aopen%20label%3A%22code%20freeze%20exception%22) for the target release version are addressed and merged
+        -    [] Validate that the stable tag in `readme.txt` in the release branch matches the stable tag in `readme.txt` in the SVN `trunk` ([here](https://plugins.svn.wordpress.org/woocommerce/trunk/readme.txt))
+        -    [] Skip changelog generation (?)
+        -    [] Create a PR to bump version in source code with `GITHUB_TOKEN=<your_PAT_token> pnpm utils code-freeze version-bump -b release/9.9 9.9.0-rc.1`
+        -    [] Review (ensure PR CI-checks are passing) and merge the PR (ensure merge CI-checks are passing)
+        -    [] Build zip file ([workflow](https://github.com/woocommerce/woocommerce/actions/workflows/release-build-zip-file.yml))
+        -    [] Validate version and stable tag in 'woocommerce.php' and `readme.txt` in the generated zip file
+
+        #### Release to WordPress.org
+
+        -    [] Skip notifying Atomic
+        -    [] Using WooRelease, perform a release dry run with e.g., `php woorelease.php wporg:release --product_version=9.9.0-rc.1 --zip_file=ZIP_FILE_PATH --product_path=plugins/woocommerce https://github.com/woocommerce/woocommerce/tree/release/9.9`
+        -    [] Using WooRelease, perform the live release with e.g., `php woorelease.php wporg:release --product_version=9.9.0-rc.1 --zip_file=ZIP_FILE_PATH --product_path=plugins/woocommerce https://github.com/woocommerce/woocommerce/tree/release/9.9 --release`
+        -    [] If the release command reports SVN-related timeouts, validate versions on https://wordpress.org/plugins/woocommerce/advanced/ and https://plugins.svn.wordpress.org/woocommerce/tags/ pages and continue if those are correct
+        -    [] Upon receiving email `[WordPress Plugin Directory] Pending release for WooCommerce`, confirm the release
+        -    [] Verify the new version is listed and downloadable on https://wordpress.org/plugins/woocommerce/advanced/ page
+
+        #### Stage and bump stable tag
+
+        -    [] Manually create a release entry on https://github.com/woocommerce/woocommerce/releases (use the version as tag, upload the generated zip, and pick the proper release branch)
+        -    [] Verify new PRs created by `github-actions` and merge ones related to the release
+
+        #### Send comms out
+
+        -    [] Coordinate with the Developer Advocacy team to update the pre-release post on developer.woocommerce.com
+        -    [] Notify #woo-core-releases and #woo-announcements Slack channels of the release with a link to the release post
```

Comment:
Can we simplify to just notify `#woo-announcements`? I'm not sure the other ping has any value, and it'll likely be automated by post-release automations soon anyways.